combine_rewards.py

```diff
@@ -22,14 +28,69 @@
 if travelers["bridge-traveler"].isna().any():
     raise ValueError("Missing values in traveler data")
 
+community = pd.DataFrame(
+    pd.read_json("final/community_rewards.json", typ="series"),
+    columns=["community"]
+)
+
 out = (
     pd.DataFrame(travelers).join(
-        [bridgoor, lp], how="outer"
+        [bridgoor, community, lp], how="outer"
     )
     .fillna(0.0)
 )
-out["total"] = out.sum(axis=1)
 out.index.name = "address"
+out.index = out.index.map(lambda x: web3.Web3.toChecksumAddress(x))
+
+# Make sure no duplicates
+assert out.index.duplicated().sum() == 0
+
+# Convert everything to decimal types for more accurate/consistent computations
+out = out.applymap(lambda x: Decimal(x).quantize(Decimal("0.0000")))
+
+# Format for mr
+mrout = {}
+mrout["recipients"] = []
+
+# Metadata
+mrout["chainId"] = 1
+mrout["rewardToken"] = ""
+mrout["windowIndex"] = 0
+
+
+# Constant 1e18 for convenience
+zeros = Decimal(1e18)
+runningSum = Decimal(0)
+
+# Iterate through rows and compute total and save to merkle tree object
+def removeDecimals(x):
+    return x.quantize(Decimal("0"))
+
+for (address, row) in out.iterrows():
+
+    # Add by hand because `.sum` converted to float
+    totalDecimal = row["bridge-traveler"] + row["bridgoor"] + row["community"] + row["lp"]
+    runningSum += totalDecimal*zeros
+    out.at[address, "total"] = totalDecimal
+
+    entry = {
+        "account": address,
+        "amount": str(removeDecimals(totalDecimal*zeros)),
+        "metadata": {
+            "amountBreakdown": {
+                "communityRewards":  str(removeDecimals(row["community"] * zeros)),
+                "welcomeTravelerRewards": str(removeDecimals(row["bridge-traveler"] * zeros)),
```

Comment:
Assume welcomeTraveler is needed for legacy reasons?

Reply:
Exactly -- This was the output I was told to replicate.